Components/ProductToShop.php

```diff
@@ -1206,4 +1209,97 @@ public function makeMainVariant($shopId, $sourceId, $groupId)
         $this->manager->persist($article);
         $this->manager->flush();
     }
+
+    /**
+     * Updates the status of an Order
+     *
+     * @param string $localOrderId
+     * @param string $orderStatus
+     * @param string $trackingNumber
+     * @return void
+     */
+    public function updateOrderStatus($localOrderId, $orderStatus, $trackingNumber)
+    {
+        if ($this->config->getConfig('updateOrderStatus') == 1) {
```

Comment:
Nögel said it should be opt-out so default is 0;
I set the default explicit in Setup now.
Do you think i have to do it in update too? 
because then zero is still the default value 
and the bootstrap/update.php doesn't have a dependency to the config component now

Reply:
Better to have it in Update as well. Until now we're doing plain SQL queries in Update.php when need to touch s_plugin_connect_config table

Reply:
Please check ConfigFactory. You can use it and add config component as a dependency in Update.php

Reply:
did it with plain SQl